apps/studio/pages/projects.tsx

```diff
@@ -11,13 +11,17 @@ import { useAutoProjectsPrefetch } from 'data/projects/projects-query'
 import { useFlag, useIsFeatureEnabled } from 'hooks'
 import { IS_PLATFORM, LOCAL_STORAGE_KEYS } from 'lib/constants'
 import { NextPageWithLayout } from 'types'
+import { useProfile } from 'lib/profile'
 
 const ProjectsPage: NextPageWithLayout = () => {
   const router = useRouter()
   const [search, setSearch] = useState('')
   const { data: organizations, isError, isSuccess } = useOrganizationsQuery()
   useAutoProjectsPrefetch()
 
+  const data = useProfile()
+  console.log(data)
+
```

Comment:
Did this slip through? ;)

Reply:
oooooooh

Reply:
thanks so much 🙏 forgot to stash the previous code changes while i was debugging something haha